### PR TITLE
Make it clearer how and what to mark as not applicable

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

Make it easier and clearer how and what to mark as not applicable in the PR Template:

- make not applicable a separate checkbox
- move the simpler items first
- make one item just an instruction, not a checkbox
- add copyright header item

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by bergmeister** [comment]
I always find that ticking the first 3 checkboxes is a pointless exercise. They should be just one checkbox stating something like `PR summary/title follows rules here` with a link to a more or less detailed document section describing it.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,15 +4,19 @@
...
+- **User-facing changes**
+    - [ ] Not Applicable
+    - **OR**
+    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
+        - [ ] Issue filed - Issue link:
+- **Testing - New and feature**
+    - [ ] Not Applicable
+    - **OR**
+    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
+        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)
```
Comment (adityapatwardhan) [inline .github/PULL_REQUEST_TEMPLATE.md]: Add another item to point to : https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission for copyright header.

**Comment by TravisEz13** [comment]
@bergmeister I agree partially on the first 3 checkboxes and updated the checklist, but just because we can't force people doesn't mean we can't remind them.  As a maintainer, the results are much better with the checklist.

The problem with using unchecked as not applicable could mean they haven't gotten around to it or that it's not applicable, but I think I would be ok with it.  I added the maintainers and Steve for additional feedback.

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

**Comment by iSazonov** [comment]
Should we add a point for [Breaking Change] in commit header? I already lost one.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -4,15 +4,20 @@
...
+    - Use the present tense and imperative mood when describing your changes
 - [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
-- [ ] User facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
-    - [ ] Issue filed - Issue link:
 - [ ] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
-- [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
-    - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)
+- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
```
Comment (adityapatwardhan) [inline .github/PULL_REQUEST_TEMPLATE.md]: Also for psd1, the author, company, copyright fields must be consistent.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (adityapatwardhan) [reply .github/PULL_REQUEST_TEMPLATE.md]: Closed.

**Comment by TravisEz13** [comment]
I think the breaking change in the commit header is a maintainer requirement.

**Review by adityapatwardhan — APPROVED** [review]
Please add psd1 to the list.

**Review by iSazonov — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -4,15 +4,20 @@
...
 - [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
     - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
+- **User-facing changes**
+    - [ ] Not Applicable
+    - **OR**
+    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
+        - [ ] Issue filed - Issue link:
+- **Testing - New and feature**
+    - [ ] Not Applicable
```
Comment (SteveL-MSFT) [inline .github/PULL_REQUEST_TEMPLATE.md]: Perhaps change this to:

Not Applicable or can only be tested interactively

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Review by SteveL-MSFT — APPROVED** [review]

**Review by daxian-dbw — APPROVED** [review]

**Pull request merged**